src/content/blog/expanding-ddev-maintainer-team.md

```diff
@@ -0,0 +1,72 @@
+
+**Onboarding and Support**: This is a wide-open Free and Open Source project. We want every contributor to have a great experience, and we want everybody to be free to contribute. But this means mentoring, and [contributor training](https://ddev.com/blog/contributor-training/). And support. Support all the time. It’s amazing how many people do not need support, but there is lots to be done every day in the [issue queue](https://github.com/ddev/ddev/issues), [Discord](https://discord.gg/hCZFfAMc5k), Slack, and [Stack Overflow](https://stackoverflow.com/tags/ddev). It’s mostly maintainers that do this support, but it’s a great community and everybody is encouraged to help.
+
+**Documentation**: You know how docs are always needing more help and they’re affected by all the factors above.
+
+### Why Do We Want More Full-time Paid Maintainers?
+
+**More Maintainers Means Better Support and Resilience**: Better support, faster responses, ability to add features. These are all things you want and we want.
+
+**Randy is not forever**: Although I love DDEV and its community we are all finite, so the time will come that I either temporarily or permanently can’t carry on. That means we need a strong maintainer presence besides me.  (In fact in the short term I plan to be bike touring in Patagonia in December 2023 and January 2024 with limited access to the internet.)
+
+**The community and its needs are growing**: More CMSs have adopted DDEV as their go-to local development environment. That means we have new and different users with new and different needs. What fun! So great! But this will eventually strain our current abilities to support.
+
+### What is DDEV’s Current Funding Situation and What are the Goals?
+
+**Current Funding**:
+
+- [**Platform.sh](http://Platform.sh)** pays Randy’s salary and provides benefits to him as an employee. That is an amazing benefit for this community and goes a long way! THANK YOU!
+- **Major sponsors [Tag1](https://tag1.com/), [1xInternet](https://1xinternet.de/), and [i-gelb](https://i-gelb.net/)** together account for USD$2000/month in funding. THANK YOU!
+- **Many other agencies and individuals** via GitHub Sponsors account for about USD$1500. THANK YOU!
+
+**Funding Goal**: Our goal is salary for a full-time paid maintainer, estimated at about USD$10,000/month or USD$120,000/year. Thanks you we’re already about 35% of the way there!
+
+### What is the Current Maintainership Situation?
+
+Currently **[Randy Fay](https://github.com/rfay), [Simon Gilli](https://github.com/gilbertsoft) and [Stas Zuk](https://github.com/stasadev)** have maintainer privileges. Simon has made enormous contributions over the years and knows how to handle most maintainer roles. He has been periodically paid by the DDEV Foundation for his work, but his available time is sometimes spotty. Stas is new to the role after many, many important contributions. Financial arrangements are pending.
```

Comment:
```suggestion
Currently **[Randy Fay](https://github.com/rfay), [Simon Gilli](https://github.com/gilbertsoft) and [Stas Zhuk](https://github.com/stasadev)** have maintainer privileges. Simon has made enormous contributions over the years and knows how to handle most maintainer roles. He has been periodically paid by the DDEV Foundation for his work, but his available time is sometimes spotty. Stas is new to the role after many, many important contributions. Financial arrangements are pending.
```

Reply:
Glad you caught that. Sorry for the typo.